lib/msgpax/plug_parser.ex

```diff
@@ -36,16 +52,26 @@ if Code.ensure_compiled?(Plug) do
       {:next, conn}
     end
 
-    def init(opts), do: opts
+    def init(options) do
+      Keyword.pop(options, :unpacker, Msgpax)
```

Comment:
Let's validate the unpacker like https://github.com/elixir-plug/plug/blob/master/lib/plug/parsers/json.ex#L52.